src/pages/workspaces/List.js

```diff
@@ -86,18 +89,57 @@ const styles = {
   })
 }
 
-const WorkspaceCard = pure(({ listView, workspace: { workspace: { namespace, name, createdBy, lastModified, attributes: { description } } } }) => {
+const WorkspaceCard = pure(({ listView, onClone, onDelete, onShare, workspace: { accessLevel, workspace: { namespace, name, createdBy, lastModified, attributes: { description } } } }) => {
   const lastChanged = `Last changed: ${Utils.makePrettyDate(lastModified)}`
   const badge = div({ title: createdBy, style: styles.badge }, [createdBy[0].toUpperCase()])
   const descText = description || span({ style: { color: colors.gray[2] } }, [
     'No description added'
   ])
+  const isOwner = _.includes(accessLevel, ['OWNER', 'PROJECT_OWNER'])
```

Comment:
This has the same name as `Utils.isOwner`, but it does something different. Should this change? Or should that?

Reply:
It seems like isOwner is largely used like the way I did throughout the code. I see one instance of the Utils.isOwner used in ShareWorkspaceModal. Perhaps the Utils name should change since it seems to be used less?

Reply:
changed to use Utils in update commit 1